packages/core/src/external/quest/fhir/condition.ts

```diff
@@ -0,0 +1,66 @@
+import _ from "lodash";
+import { uuidv7 } from "@metriport/shared/util/uuid-v7";
+import { Condition, Patient } from "@medplum/fhirtypes";
+import { ResponseDetail } from "../schema/response";
+import { ICD_10_URL, ICD_9_URL } from "@metriport/shared/medical";
+import { getPatientReference } from "./patient";
+import { getQuestDataSourceExtension } from "./shared";
+
+export function getConditions(
+  detail: ResponseDetail,
+  { patient }: { patient: Patient }
+): Condition[] {
+  const conditions = _([
+    getCondition(patient, detail.diagnosisCode1),
+    getCondition(patient, detail.diagnosisCode2),
+    getCondition(patient, detail.diagnosisCode3),
+    getCondition(patient, detail.diagnosisCode4),
+    getCondition(patient, detail.diagnosisCode5),
+    getCondition(patient, detail.diagnosisCode6),
+    getCondition(patient, detail.diagnosisCode7),
+    getCondition(patient, detail.diagnosisCode8),
+    getCondition(patient, detail.diagnosisCode9),
+    getCondition(patient, detail.diagnosisCode10),
+  ])
+    .compact()
+    .value();
+
+  return conditions;
+}
+
+function getCondition(patient: Patient, diagnosisCode?: string): Condition | undefined {
+  if (!diagnosisCode) return undefined;
+  const { system, code } = parseDiagnosisCode(diagnosisCode);
+  const extension = [getQuestDataSourceExtension()];
+
+  return {
+    resourceType: "Condition",
+    id: uuidv7(),
+    subject: getPatientReference(patient),
+    code: {
+      coding: [
+        {
+          system,
+          code,
+        },
+      ],
+    },
+    extension,
+  };
+}
```

Comment:
1. `verificationStatus` could probably?? be set to confirmed? Worth double checking how the conditions inferred from labs work though.
2. There's currently nothing tying these conditions back to the diagnostic report, they're going to be dangling and not very useful! Let's use the evidence.detail field to point them back at the observation, which i believe itself is referenced from the diagnostic report.

spec: https://hl7.org/fhir/R4/condition.html

Reply:
1. addressed with verification status generator
2. addressed with reference in evidence.detail